app/src/components/adminConsole/publishedCups/PublishedCupsHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { SectionHeader } from "../AdminConsoleStyles";
+import Elle from "../../../images/Elle";
+import styled from "styled-components";
+
+export const MiniElleImg = styled(Elle)`
+    height: 66px;
+    width: auto;
+    vertical-align: text-bottom;
+    margin-right: 19px;
+`;
```

Comment:
`MiniEllImg` is already defined three times!
Could you submit a separate PR to refactor that once this card is complete?

Reply:
This is actually part of the 'section header' reusable component, so it's already there :)